hosting_docs/source/backups_dr/4-dr-setup.rst

```diff
@@ -1,4 +1,79 @@
-Disaster Recovery Setup
-=======================
+Disaster Recovery
+=================
 
-About what DR is and what tools HQ has to support a DR setup. And a how-to guide.
\ No newline at end of file
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Overview
+--------
+
+Disaster Recovery refers to the process of restoring the IT services following an event of outage of IT infrastrcture due to a natural or human induced disaster to allow business continuity. This `public artcile <https://www.ibm.com/in-en/topics/backup-disaster-recovery>`_ from IBM is useful to understand what Disaster Recovery is in general.
+
+A Disaster Recovery solution at a minimum involves
+
+*
+  Establishing Recovery Time and Recovery Point Objectives that meet project requirements.
+
+*
+  Setting up and monitoring remote data backups or replication.
+
+*
+  Creating an active or passive secondary site and the necessary automation to failover and restore to this site rapidly.
+
+Since Commcare mobile works offline, a disaster at primary infrastructure may not cause an immediate disruption to mobile worker operations. But it definitely impacts every web operations (for e.g. actions based on reports, SMS reminders and integrations) and will soon clog mobile worker operations as well. 
```

Comment:
```suggestion
Since CommCare mobile works offline, a disaster at primary infrastructure may not cause an immediate disruption to mobile worker operations. But it definitely impacts all web operations (for e.g. actions based on reports, SMS reminders and integrations) and will soon clog mobile worker operations as well. 
```